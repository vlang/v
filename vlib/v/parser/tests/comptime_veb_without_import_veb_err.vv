fn main() {
	$veb.html('index.html')
}
