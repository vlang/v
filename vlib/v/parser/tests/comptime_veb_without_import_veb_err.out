vlib/v/parser/tests/comptime_veb_without_import_veb_err.vv:2:2: error: `$veb` cannot be used without importing veb
    1 | fn main() {
    2 |     $veb.html('index.html')
      |     ~~~~
    3 | }
